### PR TITLE
fix: datastore events model name

### PR DESCRIPTION
**Pull request by iartemiev**
*Issue #, if available:*
Fixes https://github.com/aws-amplify/docs/issues/2837

*Description of changes:*
`modelName` -> `model.name`

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/datastore/datastore-events.md [hunk 1]
```diff
@@ -52,7 +52,8 @@ HubPayload: N/A
 Dispatched when a local change has been newly staged for synchronization with the Cloud
 
 HubPayload `outboxMutationEvent` contains:
-- `modelName` (String): the name of the model that is awaiting publication to the Cloud
+- `model`:
+    - `name` (String): the name of the model that was synced
```
Comment (amhinson) [inline docs/lib/datastore/datastore-events.md:56]: Did you mean to replace the descriptions here and for the one below?

Comment (iartemiev) [reply docs/lib/datastore/datastore-events.md:56]: Bad copypasta 🍝.. Good catch!

Comment (iartemiev) [reply docs/lib/datastore/datastore-events.md:56]: Fixed!

**Review by amhinson — APPROVED** [review]
LGTM 👍

**Review by richardmcclellan — CHANGES_REQUESTED** [review]
I believe this will break the documentation for iOS and Android, because they both have a `modelName` property, not `model.name`.  Would it be reasonable to add a modelName property to the JS event for platform consistency?   If not, please make sure this docs update only affects JS.

**Comment by iartemiev** [comment]
> Instead of this docs change, would it be reasonable to add a `modelName` property to the JS `OutboxMutationEvent` for platform consistency? 

Yea, I think that makes more sense. I don't see an issue with that. Will confirm and then close this PR in favor of adding that property to JS.

**Comment by iartemiev** [comment]
> Instead of this docs change, would it be reasonable to add a `modelName` property to the JS `OutboxMutationEvent` for platform consistency? 

@richardmcclellan, yea, I think that makes more sense. I don't see an issue with that. Will confirm and then close this PR in favor of adding that property to JS.

**Pull request closed without merging**